data/templates/install-coreos.sh

```diff
@@ -2,4 +2,5 @@
 set -e
 curl -O http://<%=server%>:<%=port%>/api/current/templates/pxe-cloud-config.yml
 sudo coreos-install -d <%=installDisk%> -c pxe-cloud-config.yml -b <%=repo%>
+wget --spider http://<%=server%>:<%=port%>/api/current/templates/<%=completionUri%>
```

Comment:
not sure you want to use `--spider` here unless you're really trying to spider through that directory - the end result if you passed in a `completionUri` that represented a directory would be replicating that entire directory structure down (probably inadvertently) to the local disk at the root of the CoreOS setup.

I'm also not sure if the CoreOS install includes `wget`, although I know it includes curl. You might just instead have this be:

`curl -s http://<%=server%>:<%=port%>/api/current/templates/<%=completionUri%> >> /dev/null` 

so you quite out the logging for curl and explicitly throw away the results